.azure/pipelines/quarantined-pr.yml

```diff
@@ -1,9 +1,46 @@
-# We want to run quarantined tests on master as well as on PRs
+#
+# See https://docs.microsoft.com/en-us/vsts/pipelines/yaml-schema for details on this file.
+#
+
+# Configure which branches trigger builds
+# We want to run quarantined tests on release/5.0 and master as well as on PRs
 trigger:
   batch: true
   branches:
     include:
     - master
+    - release/5.0
```

Comment:
Does anyone remember if this needs to be checked into release/5.0 for AzDO to pay attention❔

Reply:
I don't think so, @mmitche can you confirm?

Reply:
Yes, triggers are read from the target branch.

Reply:
@Pilchie would you like to have rolling builds of the quarantined pipeline for 5.0.1❔

Similar question for the 6 scheduled builds per day if we want to continue with that. Those also focus entirely on the master branch.

Reply:
Yes, I think it makes sense to have them for 5.0.1.  Especially the scheduled ones, as that's the way we get more platform coverage - I would hate to check in a servicing fix that breaks on some platforms and not know because we didn't enable the tests.

Reply:
To clarify: This is the pipeline which runs for every PR. It also does rolling builds and runs every four hours but just for master. My question here is whether we should update aspnetcore-quarantined-pr to perform rolling builds of release/5.0. [Separately, I'm dubious](https://github.com/dotnet/aspnetcore/pull/27484#discussion_r517028937) about the addition six builds we run of this pipeline.

Adding release/5.0 builds of aspnetcore-helix-matrix and aspnetcore-quarantined-tests doubles the resources consumed and might imply we should reduce aspnetcore-helix-matrix to once (instead of twice) per day. Those pipelines gobble a lot of machines. aspnetcore-quarantined-tests already runs once per day but both (unfortunately) run only for the master branch. Thoughts❔

Reply:
Fair point on the scheduled runs - I don't think we need them for servicing branches.  We had them for master to get enough data to judge the flakiness of tests.

As for the aspnetcore-helix-matrix runs, I strongly believe that we should run those tests for servicing branches, but perhaps less often there.  I suggest once/day only if changes have been checked into the servicing branch.

Reply:
:ok: I'll do a follow-up PR in the release/5.0 branch to add a build per day of aspnetcore-helix-matrix. I'll include the above change to add rolling builds of the aspnetcore-quarantined-pr pipeline.

 I'm not going to cherry-pick the rest of this PR into release/5.0. Doesn't seem worthwhile since we very rarely (if ever) update just docs while servicing.